cmd/livepeer/livepeer.go

```diff
@@ -58,7 +58,8 @@ func main() {
 	rtmpAddr := flag.String("rtmpAddr", "127.0.0.1:"+RtmpPort, "Address to bind for RTMP commands")
 	cliAddr := flag.String("cliAddr", "127.0.0.1:"+CliPort, "Address to bind for  CLI commands")
 	httpAddr := flag.String("httpAddr", "", "Address to bind for HTTP commands")
-	transcoder := flag.Bool("transcoder", false, "Set to true to be a transcoder")
+	orchAndTrans := flag.Bool("transcoder", false, "Set to true to be a transcoder")
```

Comment:
does this mean we currently can't start the node only as a transcoder? - if so, could you please share why that's the case?

thanks!

Reply:
We can start as a transcoder only. Most of the thinking behind these flags was outlined in https://github.com/livepeer/go-livepeer/issues/567.

As of this PR, there are three distinct modes of operation:

* Orchestrator + (local) transcoder . This is the old behavior, and it's preserved with the `-transcoder` flag. We aren't breaking anyone's setups this way.
* Standalone orchestrator. This is introduced with the `-orchestrator` / `-orchSecret` flag.
*  Standalone (remote) transcoder. This is indicated by the `-orchAddr` / `-orchSecret` flag where `orchAddr` points to an orchestrator that's somewhere else.